docs/source/upgrade-guide/index.md

```diff
@@ -132,6 +132,13 @@ the block title. Before, it took the `id` of the block, which is utterly wrong a
 Following the same convention as the above change, `Variation` field coming from the block enhancers now uses the `title` of the block as source for translating
 the variation title. Before, it took the `id` of the block, which as stated before, is wrong and missleading. There is a chance that  this change will trigger untranslated variation titles in your projects and add-ons.
 
+### Listing block no longer retrieve fullobjects
```

Comment:
@sneridagh can we add a section to the regular docs that explains the differences between fullobjects and brains? Or is that something that is already fully documented on plone.restapi?